SPECS/crash/crash-printk-fix.patch

```diff
@@ -0,0 +1,452 @@
+This patch is a combination of the following commits:
```

Comment:
How difficult would it be to apply as individual 2 patches? It would be nice to preserve the upstream commit information and annotate if we've made minor changes to get the commit to apply.

Reply:
The second commit edits a file introduced in the first commit. So I chose to combine them (by applying both in order and then generating a diff of everything). Since there were no changes otherwise, and this has already been tested, I'd prefer to keep as-is.